src/openforms/contrib/kadaster/api/views.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import re
+
+from django.conf import settings
+from django.http import HttpResponseBadRequest
+from django.utils.translation import gettext_lazy as _
+
+import requests
+from djangorestframework_camel_case.render import CamelCaseJSONRenderer
+from drf_spectacular.types import OpenApiTypes
+from drf_spectacular.utils import OpenApiParameter, extend_schema
+from rest_framework import status
+from rest_framework.generics import GenericAPIView
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from openforms.api.serializers import ExceptionSerializer, ValidationErrorSerializer
+from openforms.submissions.api.permissions import AnyActiveSubmissionPermission
+
+from .serializers import MapSearchSerializer
+
+logger = logging.getLogger(__name__)
+
+
+@extend_schema(
+    summary=_("List BAG address suggestions."),
+    parameters=[
+        OpenApiParameter(
+            "q",
+            OpenApiTypes.STR,
+            OpenApiParameter.QUERY,
+            description=_("The search query we send to the pdok locatieserver api."),
+            required=True,
+        )
+    ],
+    responses={
+        200: MapSearchSerializer,
```

Comment:
```suggestion
        200: MapSearchSerializer(many=True),
```

The API documentation now makes it look like a single object is returned instead of an array of objects.